test/sdp.spec.ts

```diff
@@ -53,6 +53,11 @@ describe('SDP', () => {
     ])
   })
 
+  it('extracts a fingerprint from sdp', () => {
```

Comment:
we should add a negative test here to validate the fallback behaviour

Reply:
this just tests the regex behaviour. While there are no explicit tests for the fallback behaviour, it gets tested when the tests are run in Firefox as part of CI.